src/goto-programs/lazy_goto_model.cpp

```diff
@@ -158,8 +159,8 @@ void lazy_goto_modelt::initialize(const cmdlinet &cmdline)
 
     if(language_files.typecheck(symbol_table))
     {
-      msg.error() << "CONVERSION ERROR" << messaget::eom;
-      throw 0;
+      // TODO more helpful error message
```

Comment:
I'm not sure this is a TODO (and neither in the parser case): those "more helpful error messages" are provided already, during parsing/type checking.